cub/test/catch2_test_device_histogram.cu

```diff
@@ -436,7 +436,7 @@ C2H_TEST("DeviceHistogram::Histogram* large levels", "[histogram][device]", c2h:
   using sample_t             = c2h::get<0, TestType>;
   using level_t              = sample_t;
   const auto max_level_count = 128;
-  auto max_level             = cub::NumericTraits<level_t>::Max();
+  auto max_level             = cuda::std::numeric_limits<level_t>::max();
```

Comment:
NItpick: 
```suggestion
  auto max_level             = ::cuda::std::numeric_limits<level_t>::max();
```

Reply:
I did fully qualify in the past, but at some point @elstehle argued, that it's not necessary in the unit tests. I don't care. Always qualifying is easier, since I don't need to track inside which source file I am.